src/frontend/screens/Track/index.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import {
+  StyleSheet,
+  View,
+  Text,
+  SafeAreaView,
+  ScrollView,
+  Pressable,
+} from 'react-native';
+import {BLACK, WHITE, DARK_GREY} from '../../lib/styles.ts';
+
+import TrackIcon from '../../images/Track.svg';
+import EditIcon from '../../images/Edit.svg';
+import {CustomHeaderLeft} from '../../sharedComponents/CustomHeaderLeft.tsx';
+import {defineMessages} from 'react-intl';
+import {useDeleteTrackMutation, useTrack} from '../../hooks/server/track.ts';
+import {useObservations} from '../../hooks/server/observations.ts';
+import {NativeNavigationComponent} from '../../sharedTypes/navigation';
+import {MapPreview} from './MapPreview.tsx';
+import {ObservationList} from './ObservationList.tsx';
+import {ErrorBottomSheet} from '../../sharedComponents/ErrorBottomSheet.tsx';
+import {ActionButtons} from '../../sharedComponents/ActionButtons.tsx';
+
+const m = defineMessages({
+  title: {
+    id: 'screens.Track.title',
+    defaultMessage: 'Track',
+    description:
+      'Title of track screen showing (non-editable) view of observation with map',
+  },
+  deleteTitle: {
+    id: 'screens.Track.deleteTitle',
+    defaultMessage: 'Delete track?',
+    description: 'Title of dialog asking confirmation to delete a track',
+  },
+});
+
+export const TrackScreen: NativeNavigationComponent<'Track'> = ({
```

Comment:
https://github.com/digidem/comapeo-mobile/pull/412/commits/683572e4c0eeeff1b83f61a889fb0c44fb73a4f7

Reply:
hm there's kind of an odd consequence of switching to the component i suggested: now the header sticks to the top and overlays the content as you scroll down, which kind of feels odd 🤔 don't think it's something that needs to be addressed here but maybe it's a question to bring to design.

<img width="200" alt="image" src="https://github.com/digidem/comapeo-mobile/assets/18542095/05180c88-74f0-4eb1-98a9-9c2c9f8df840">

part of me thinks that maybe it would be better to have the normal navigation header such that it's not transparent and absolutely positioned (i.e. undoing the header nav option i suggested before)